src/utilities/formInputConfigs.js

```diff
@@ -55,6 +55,7 @@ const FORM_INPUT_CONFIGS = seedObject => ({
     isRequired: true,
     label: formInputStrings.first_name,
     invalidMessage: formInputStrings.must_not_be_empty,
```

Comment:
I can't help thinking that `isReadOnly` would have been much easier!
Would default to `false` and so only two lines need to be added

Reply:
Hmm, fair point. This is a bit of a quick fix and likely to be something revisited later anyway, so nothing finalised yet!